docker-compose.yml

```diff
@@ -0,0 +1,16 @@
+version: '3.4'
+
+services:
+  db:
+    image: postgres
+    volumes:
+      - ./tmp/db:/var/lib/postgresql/data
+  app:
+    build: .
+    command: bash -c "rm -f tmp/pids/server.pid && bundle exec rails s -p 3000 -b '0.0.0.0'"
```

Comment:
if you add `tmp/pids/server.pid` to `.dockerignore` you won't get this file in the first place